public/app/features/commandPalette/actions/useDashboardResults.ts

```diff
@@ -0,0 +1,23 @@
+import debounce from 'debounce-promise';
+import { useEffect, useState } from 'react';
+
+import { CommandPaletteAction } from '../types';
+
+import { getDashboardSearchResultActions } from './dashboardActions';
+
+const debouncedDashboardSearch = debounce(getDashboardSearchResultActions, 200);
+
+export const useDashboardResults = (searchQuery: string, isShowing: boolean) => {
```

Comment:
nitpicky, but i think it's fine for this to just go in the dashboardActions file.